### DISCUSSION TITLE
ai写的告白气球。

### DISCUSSION THREAD

**Discussion by hdjsjjdgrhj** [share121/inter-knot#5044] [其他]
居然还挺好听？

Uploading 告白气球.mp4…

**Comment by sb35386057** [comment]
Mp4链接呢？

**Comment by hdjsjjdgrhj** [comment]
这里。
https://github.com/user-attachments/assets/405446fe-9fb1-42f6-88d1-cfedff2a8c2c

**Reply by sb35386057** [reply]
确实豪，副歌你的眼睛那4个字抬高跟原歌大同小异，不过我我咋没见你在歌词里调分段一分段二还有副歌段的，你是怎么控制唱副歌的时候唱副歌的🤔
